webapp/src/main/webapp/news-publish-targets-management/components/NewsPublishTargetsManagementDrawer.vue

```diff
@@ -97,14 +107,68 @@ export default {
         return '';
```

Comment:
transform to return this.targetName && !this.targetName.trim().match(/^[\w\-\s]+$/) && this.targetName.length > 0 ? this.$t('news.list.settings.name.errorMessage') : ''